README.md

```diff
@@ -26,7 +26,7 @@ Curated list of Uzbek Telegram dev groups
 - [@uzverstalshiki](https://t.me/uzverstalshiki)
 - [@vuejs_uz](https://t.me/vuejs_uz)
 - [@react_uz](https://t.me/react_uz)
-- [@Next_JS_uz](https://t.me/next_js_uz)
```

Comment:
@mcpeblocker What happened with this one?

Reply:
> @mcpeblocker What happened with this one?

The previous was not active and well-maintained, a while ago I found the new one and its maintainers seemed to be active and helpful.